### PR TITLE
class_mfaAccount.inc: Make ACLs more clearer + organized.

### PR DESCRIPTION
**Pull request by dzatoah**
+ Mark 'r' and 'w' of every ACL.

### CODE REVIEW

File: personal/privacyidea/class_mfaAccount.inc [hunk 1]
```diff
@@ -935,23 +935,23 @@ class mfaAccount extends plugin
...
+                "addTokenPaper"         => _("Add TAN list")                  . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenRegistration"  => _("Add Recovery key")              . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenWebAuthn"      => _("Add Security key")              . "&nbsp;(" . _("write only") . ") ", # w
+                "addTokenTotp"          => _("Add Authentication app (TOTP)") . "&nbsp;(" . _("write only") . ") ", # w
+                "tokenDescription"      => _("Description"),                              #rw
+                "tokenLastUsed"         => _("Last use"),                                 #rw
+                "tokenFailCount"        => _("Failed login counter"),                     #rw
+                "tokenStatus"           => _("Status"),                                   #rw
+                "tokenCountAuth"        => _("Authentication count"),                     #rw
+                "tokenCountAuthSuccess" => _("Successful authentication count"),          #rw
```
Comment (sunweaver) [inline personal/privacyidea/class_mfaAccount.inc:952]: Please mark (as already discussed in our last video call) the ACLs that are theoretically read-write in GOsa², but read-only by PI API.

**Review by sunweaver — CHANGES_REQUESTED** [review]
See inline comment.

File: personal/privacyidea/class_mfaAccount.inc (hunk 1 above)
Comment (dzatoah) [reply personal/privacyidea/class_mfaAccount.inc]: Please have a look again.

**Review by sunweaver — APPROVED** [review]
LGTM

**Pull request merged**